front/app/containers/Admin/users/ChangeSeatModal.tsx

```diff
@@ -77,69 +75,85 @@ const ChangeSeatModal = ({
+        {formatMessage(modalTitle)}
+      </Text>
+    </Box>
+  ) : undefined;
 
   return (
-    <Modal
-      opened={showModal}
-      close={closeModal}
-      header={
-        <Box px="2px">
-          <Text color="primary" my="8px" fontSize="l" fontWeight="bold">
-            {formatMessage(modalTitle)}
-          </Text>
-        </Box>
-      }
-    >
-      <Box display="flex" flexDirection="column" width="100%" p="32px">
-        <Box>
-          <Text color="textPrimary" fontSize="m" my="0px">
-            <FormattedMessage
-              {...confirmChangeQuestion}
-              values={{
-                name: (
-                  <Text as="span" fontWeight="bold" fontSize="m">
-                    {`${userToChangeSeat.attributes.first_name} ${userToChangeSeat.attributes.last_name}`}
-                  </Text>
-                ),
-              }}
-            />
-          </Text>
-          <Box py="32px">
-            <SeatInfo seatType="admin" />
-          </Box>
-        </Box>
-        <Box
-          display="flex"
-          flexDirection="row"
-          width="100%"
-          alignItems="center"
-        >
-          <Button
-            width="auto"
-            onClick={() => {
-              toggleAdmin();
+    <>
+      <Modal opened={showModal} close={closeModal} header={header}>
+        {showSuccess ? (
+          <SeatSetSuccess
+            closeModal={() => {
               closeModal();
+              setShowSuccess(false);
             }}
-          >
-            {formatMessage(buttonText)}
-          </Button>
-        </Box>
-      </Box>
-    </Modal>
+            hasExceededSetSeats={hasExceededSetSeats}
+            seatType="admin"
+          />
+        ) : (
+          <Box display="flex" flexDirection="column" width="100%" p="32px">
+            <Box>
+              <Text color="textPrimary" fontSize="m" my="0px">
+                <FormattedMessage
+                  {...confirmChangeQuestion}
+                  values={{
+                    name: (
+                      <Text as="span" fontWeight="bold" fontSize="m">
+                        {`${userToChangeSeat.attributes.first_name} ${userToChangeSeat.attributes.last_name}`}
+                      </Text>
+                    ),
+                  }}
+                />
+              </Text>
+              <Box py="32px">
+                <SeatInfo seatType="admin" />
+              </Box>
+            </Box>
+            <Box
+              display="flex"
+              flexDirection="row"
+              width="100%"
+              alignItems="center"
+            >
+              <Button
+                width="auto"
+                onClick={() => {
+                  toggleAdmin();
+                  if (!isUserAdmin) {
+                    setShowSuccess(true);
+                  } else {
+                    closeModal();
+                  }
+                }}
+              >
```

Comment:
I found                   
`if (!isUserAdmin) { 
`hard to understand here.

Could we create an extra var that says "userToChangeSeatBecameAdmin" or similar?

Reply:
Comment should do. I'll add one